octavia_f5/restclient/as3objects/service.py

```diff
@@ -176,7 +175,7 @@ def get_service(listener, cert_manager, esd_repository):
                 entities.append((name, irule))
 
         # Pool member certificate handling (TLS backends)
-        if pool.tls_enabled:
+        if pool.tls_enabled and listener.protocol not in [ const.PROTOCOL_TCP, const.PROTOCOL_UDP, const.PROTOCOL_HTTPS ]:
```

Comment:
Please use a in [ .. ] condition instead of not in [ ... ] to make the condition clearer.